vendor/plugins/edu.harvard.dce.paella.presentationOnlyPlugin/presentationOnlyPlugin.js

```diff
@@ -0,0 +1,290 @@
+// PresentationOnlyPlugin toggle purpose: Turn off presenter source to reduce bandwidth when presentation only view.
+// One activation on qualities change (called directly).
```

Comment:
FYI - This plugin is not enabled in our 6x upgrade. It was used in Paella5x. In Paella6x we're using the default Paella qualities plugin